<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>G_Tours</class>
 <widget class="QMainWindow" name="G_Tours">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tableWidgetTours">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>801</width>
      <height>551</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>801</width>
      <height>551</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>801</width>
      <height>551</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QWidget {
    background-color: #1A3D7C; /* Bleu foncé inspiré du football */
    border: 2px solid #FFD700; /* Doré pour une touche premium */
}

QLabel {
    color: black;
    font-size: 14px;
    font-weight: bold;
}

QTabWidget::pane {
    border: 2px solid #FFD700;
    background: #1A3D7C;
    border-radius: 5px;
}

QTabBar::tab {
    background: #0E2A58;
    color: white;
    padding: 8px;
    border: 1px solid #FFD700;
    border-top-left-radius: 5px;
    border-top-right-radius: 5px;
}

QTabBar::tab:selected {
    background: #FFD700;
    color: #1A3D7C;
    font-weight: bold;
}

QPushButton {
    background-color: #FFD700;
    color: #1A3D7C;
    border-radius: 5px;
    padding: 8px;
    font-size: 14px;
    font-weight: bold;
}

QPushButton:hover {
    background-color: #FFC107;
}

QLineEdit, QTextEdit {
    background-color: white;
    border: 1px solid #FFD700;
    padding: 5px;
    border-radius: 3px;
}
</string>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>ADD TOUR</string>
     </attribute>
     <widget class="QPushButton" name="pushButton1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>390</y>
        <width>192</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #2E7D32; /* Vert foncé, style football */
    color: white;
    font-size: 14px;
    font-weight: bold;
    padding: 10px;
    border-radius: 5px;
    border: 2px solid #1B5E20; /* Vert plus foncé pour une touche premium */
    transition: background-color 0.3s, border-color 0.3s;
}

QPushButton:hover {
    background-color: #43A047; /* Vert plus clair pour l'effet hover */
    border-color: #388E3C;
}

QPushButton:pressed {
    background-color: #1B5E20; /* Vert très foncé pour effet de pression */
    border-color: #145A32;
}
</string>
      </property>
      <property name="text">
       <string>add</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="linetourname">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>20</y>
        <width>431</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color: #ffffff;  /* White background */
    color: #333333;             /* Dark gray text */
    border: 1px solid #cccccc;  /* Light gray border */
    border-radius: 5px;         /* Rounded corners */
    padding: 10px;              /* Padding for inner space */
    font: 14px &quot;Segoe UI&quot;, sans-serif; /* Modern font */
    transition: all 0.3s ease;  /* Smooth transition for all visual changes */
}

QLineEdit:hover {
    border-color: #007bff;      /* Blue border on hover */
}

QLineEdit:focus {
    border-color: #007bff;      /* Blue border on focus */
    box-shadow: 0 0 5px rgba(0, 123, 255, 0.6); /* Soft blue glow effect */
    background-color: #f7f7f7;  /* Light gray background on focus */
}

QLineEdit:disabled {
    background-color: #f0f0f0;  /* Light gray background when disabled */
    color: #999999;              /* Light gray text when disabled */
    border-color: #dddddd;      /* Light border color */
    cursor: not-allowed;        /* Disabled cursor */
}

QLineEdit::placeholder {
    color: #999999;              /* Placeholder text in light gray */
    font-style: italic;          /* Italic placeholder text */
}
</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>170</y>
        <width>431</width>
        <height>28</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>120</y>
        <width>431</width>
        <height>28</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Quatre Final</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Demi Final</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Final</string>
       </property>
      </item>
     </widget>
     <widget class="QDateEdit" name="dateEdit">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>270</y>
        <width>431</width>
        <height>29</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="dateEdit_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>320</y>
        <width>431</width>
        <height>29</height>
       </rect>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>390</y>
        <width>271</width>
        <height>85</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTextEdit {
    background-color: white;
    color: black;  /* You can change the text color if needed */
}
</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>TOURS NAME :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>170</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>TOUR_LEVEL</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>120</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>ROUNDS :</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>270</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>start_date</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>320</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>end_date</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>420</y>
        <width>151</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>descreption</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>70</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>TOUR_location :</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="linetourname_4">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>70</y>
        <width>431</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    background-color: #ffffff;  /* White background */
    color: #333333;             /* Dark gray text */
    border: 1px solid #cccccc;  /* Light gray border */
    border-radius: 5px;         /* Rounded corners */
    padding: 10px;              /* Padding for inner space */
    font: 14px &quot;Segoe UI&quot;, sans-serif; /* Modern font */
    transition: all 0.3s ease;  /* Smooth transition for all visual changes */
}

QLineEdit:hover {
    border-color: #007bff;      /* Blue border on hover */
}

QLineEdit:focus {
    border-color: #007bff;      /* Blue border on focus */
    box-shadow: 0 0 5px rgba(0, 123, 255, 0.6); /* Soft blue glow effect */
    background-color: #f7f7f7;  /* Light gray background on focus */
}

QLineEdit:disabled {
    background-color: #f0f0f0;  /* Light gray background when disabled */
    color: #999999;              /* Light gray text when disabled */
    border-color: #dddddd;      /* Light border color */
    cursor: not-allowed;        /* Disabled cursor */
}

QLineEdit::placeholder {
    color: #999999;              /* Placeholder text in light gray */
    font-style: italic;          /* Italic placeholder text */
}
</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>220</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>status :</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_7">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>220</y>
        <width>431</width>
        <height>28</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>active</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>non active</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>LISTE_TOUR</string>
     </attribute>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>420</y>
        <width>83</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton *championsButton = new QPushButton(&quot;Champions League&quot;, rightPanel);
championsButton-&gt;setGeometry(50, 50, 200, 50); // Position and size of the button

// Applying the stylesheet for the button
championsButton-&gt;setStyleSheet(
    &quot;background-color: #005bbb; &quot;
    &quot;color: white; &quot;
    &quot;font: bold 14px; &quot;
    &quot;padding: 10px 20px; &quot;
    &quot;border: 2px solid #003366; &quot;
    &quot;border-radius: 12px; &quot;
    &quot;text-align: center; &quot;
    &quot;box-shadow: 0 4px 8px rgba(0, 0, 0, 0.3);&quot;
    &quot;QPushButton:hover { &quot;
    &quot;background-color: #003366; &quot;
    &quot;border: 2px solid #005bbb; &quot;
    &quot;} &quot;
    &quot;QPushButton:pressed { &quot;
    &quot;background-color: #001f3d; &quot;
    &quot;border: 2px solid #003366; &quot;
    &quot;box-shadow: 0 2px 4px rgba(0, 0, 0, 0.5); &quot;
    &quot;} &quot;
    &quot;QPushButton:focus { &quot;
    &quot;outline: none; &quot;
    &quot;}&quot;
);
</string>
      </property>
      <property name="text">
       <string>update</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>370</y>
        <width>83</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton *championsButton = new QPushButton(&quot;Champions League&quot;, rightPanel);
championsButton-&gt;setGeometry(50, 50, 200, 50); // Position and size of the button

// Applying the stylesheet for the button
championsButton-&gt;setStyleSheet(
    &quot;background-color: #005bbb; &quot;
    &quot;color: white; &quot;
    &quot;font: bold 14px; &quot;
    &quot;padding: 10px 20px; &quot;
    &quot;border: 2px solid #003366; &quot;
    &quot;border-radius: 12px; &quot;
    &quot;text-align: center; &quot;
    &quot;box-shadow: 0 4px 8px rgba(0, 0, 0, 0.3);&quot;
    &quot;QPushButton:hover { &quot;
    &quot;background-color: #003366; &quot;
    &quot;border: 2px solid #005bbb; &quot;
    &quot;} &quot;
    &quot;QPushButton:pressed { &quot;
    &quot;background-color: #001f3d; &quot;
    &quot;border: 2px solid #003366; &quot;
    &quot;box-shadow: 0 2px 4px rgba(0, 0, 0, 0.5); &quot;
    &quot;} &quot;
    &quot;QPushButton:focus { &quot;
    &quot;outline: none; &quot;
    &quot;}&quot;
);
</string>
      </property>
      <property name="text">
       <string>delete</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="linetourname_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>140</y>
        <width>781</width>
        <height>192</height>
       </rect>
      </property>
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>8</number>
      </property>
      <column>
       <property name="text">
        <string>tour id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>tour name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string/>
       </property>
      </column>
      <column>
       <property name="text">
        <string/>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Rounds</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>End Date</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Start Date</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Status</string>
       </property>
      </column>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>10</y>
        <width>411</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QLineEdit {
    border: 2px solid #0077cc;  /* Bleu professionnel */
    border-radius: 8px;         /* Coins arrondis */
    padding: 6px 12px;          /* Espacement interne */
    font-size: 14px;            /* Taille du texte */
    color: #333;                /* Couleur du texte */
    background-color: #f8f8f8;  /* Fond légèrement gris */
    selection-background-color: #0077cc; /* Couleur de sélection */
}

/* Effet au survol */
QLineEdit:hover {
    border: 2px solid #005fa3; /* Bleu plus foncé */
}

/* Effet quand on clique (focus) */
QLineEdit:focus {
    border: 2px solid #004080;
    background-color: white;
}
</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Rechercher :</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioName">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>50</y>
        <width>112</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioRounds">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>50</y>
        <width>112</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Rounds</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioLevel">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>50</y>
        <width>112</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Level</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_exportPDF">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>390</y>
        <width>101</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #2E7D32; /* Vert foncé, style football */
    color: white;
    font-size: 14px;
    font-weight: bold;
    padding: 10px;
    border-radius: 5px;
    border: 2px solid #1B5E20; /* Vert plus foncé pour une touche premium */
    transition: background-color 0.3s, border-color 0.3s;
}

QPushButton:hover {
    background-color: #43A047; /* Vert plus clair pour l'effet hover */
    border-color: #388E3C;
}

QPushButton:pressed {
    background-color: #1B5E20; /* Vert très foncé pour effet de pression */
    border-color: #145A32;
}
</string>
      </property>
      <property name="text">
       <string>pdf</string>
      </property>
      <property name="icon">
       <iconset theme="QIcon::ThemeIcon::MailAttachment"/>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Statistiques</string>
     </attribute>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>90</y>
        <width>121</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>TOURS ACTIVE </string>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdActive">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>160</y>
        <width>201</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QLCDNumber" name="lcdNonActive">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>160</y>
        <width>201</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>90</y>
        <width>161</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>TOURS NON ACTIVE </string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>440</y>
        <width>171</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: #2E7D32; /* Vert foncé, style football */
    color: white;
    font-size: 14px;
    font-weight: bold;
    padding: 10px;
    border-radius: 5px;
    border: 2px solid #1B5E20; /* Vert plus foncé pour une touche premium */
    transition: background-color 0.3s, border-color 0.3s;
}

QPushButton:hover {
    background-color: #43A047; /* Vert plus clair pour l'effet hover */
    border-color: #388E3C;
}

QPushButton:pressed {
    background-color: #1B5E20; /* Vert très foncé pour effet de pression */
    border-color: #145A32;
}
</string>
      </property>
      <property name="text">
       <string>statistique</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>Tour Ratings</string>
     </attribute>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>160</y>
        <width>21</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>160</y>
        <width>21</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_3">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>160</y>
        <width>21</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_4">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>160</y>
        <width>21</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_5">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>160</y>
        <width>21</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>160</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>Submit Rating</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>75</x>
        <y>270</y>
        <width>381</width>
        <height>192</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Tour Name </string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Average Rating</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Number of Ratings   </string>
       </property>
      </column>
     </widget>
     <widget class="QComboBox" name="comboBox_3">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>80</y>
        <width>311</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
